include/Oscillator.h

```diff
@@ -28,7 +28,6 @@
 
 #include <cassert>
 #include <fftw3.h>
-#include <math.h>
 
 #ifdef LMMS_HAVE_STDLIB_H
 #include <stdlib.h>
```

Comment:
There still seem to be a few of these C headers lying around.

Reply:
Indeed. I did it only for the CPP files. Will be fixed with next push.